src/server/rpc/procedures/earthquakes/earthquakes.js

```diff
@@ -61,12 +61,27 @@ Earthquakes._sendNext = function(socket) {
     }
 };
 
+/**
+ * Stop sending earthquake messages
+ */
 Earthquakes.stop = function() {
     var uuid = this.socket.uuid;
     delete Earthquakes._remainingMsgs[uuid];
     return '';
 };
 
+
+/**
+ * Get the current price of the given stock, with a 15 min delay
+ * @param {Latitude=} minLatitude Minimum latitude of region
+ * @param {Latitude=} maxLatitude Maximum latitude of region
+ * @param {Longitude=} minLongitude Minimum longitude of region
+ * @param {Longitude=} maxLongitude Maximum longitude of region
```

Comment:
Although these are technically optional based on the implementation, using the defaults will not be very useful. It might be worth removing the optional notation from these args and then removing the casting from the implementation.

Reply:
Did both changes. The example for it doesn't seem to ever use those without values, so we should be fine.